src/auth/jwt/jwt.token.with.jku.processor.ts

```diff
@@ -32,13 +32,12 @@ export class JwtTokenWithJKUProcessor extends JwtTokenProcessor {
   async createToken(payload: jose.JWTPayload): Promise<string> {
     this.log.debug('Call createToken');
     const pkcs8 = await jose.importPKCS8(this.key, 'RS256');
-    const jwt = await new jose.SignJWT(payload)
+    return await new jose.SignJWT(payload)
```

Comment:
`await` statement is redundant 😄 

```suggestion
    return new jose.SignJWT(payload)
```